src/main/kotlin/racingcar/Car.kt

```diff
@@ -1,8 +1,10 @@
 package racingcar
 
-data class Car(val position: Int = DEFAULT_POSITION) {
+data class Car(val name: String, val position: Int = DEFAULT_POSITION) {
 
-    fun move(isMove: Boolean): Car = if (isMove) Car(position + PLUS_POSITION) else this
+    fun move(isMove: Boolean): Car = if (isMove) Car(this.name, position + PLUS_POSITION) else this
```

Comment:
```suggestion
    fun move(isMove: Boolean): Car = if (isMove) {
        this.copy(position = position + PLUS_POSITION)
    } else {
        this
    }
```

- Postion이 변경 될 때마다 이름까지 모두 새로 넣는 것보다 data class의 copy()를 통하여 특정 부분만 변경해주는게 좋다고 생각합니다.
- if절에 대한 코드 블럭에 대해서는 저는 명확하게 표현해주고 있습니다.

Reply:
특정 부분만 변경할 수 있는 copy 가 자바에선 마치 부 생성자를 연상하게 하네요 :)
자바에서의 clone 과 copy가 유사하다고 생각했는데 오히려 생성자와 비슷한 느낌이네요 감사합니다~